src/smartCar/Webserver.h

```diff
@@ -1,9 +1,43 @@
 #ifndef WEBSERVER_H
 #define WEBSERVER_H
 
+
+#include <iostream>
+#include <HTTP_Method.h>
+#include <WebServer.h>
 #include <WiFi.h>
+#include <ESPmDNS.h>
+
+class Webserver {
+
+  private:
+
+//VARIABLES
+const char* ssid;
+const char* password;
+WebServer server();
```

Comment:
You probably should specify the argument. I'd suggest something like `WebServer server{80};`